### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by RockEater** [Heroic-Games-Launcher/HeroicGamesLauncher#904] [Q&A] [answered]
I have just started to use Linux so I am not aware how updates supposed to work.
What is the process to update the Appimage, and what about the .deb file?

Thank you

**Comment by flavioislima** [comment, marked as answer]
For now, both needs to be downloaded again. For the appimage, just replace the file and for the deb you just install it again.
We will have auto update for AppImages in the future. We are working on it.

**Reply by RockEater** [reply]
Thanks
